src/connector/src/source/kafka/source/reader.rs

```diff
@@ -100,6 +100,10 @@ impl SplitReader for KafkaSplitReader {
             .await
             .context("failed to create kafka consumer")?;
 
+        // poll consumer to trigger callback functions
+        // We don't need to wait for the result, because recv may block and wait for next new message
+        let _ = tokio::time::timeout(Duration::from_millis(100), consumer.recv()).await;
```

Comment:
Is it because we call `recv` before `assign` (in L155) that `recv` becomes a function purely for side-effect (trigger the callback) without fetching any message?

Reply:
Also, IIUC `consumer.recv` will be blocked indefinitely here. how do we make sure that the callback function must be triggered before the 100ms timeout?